metrics/network/network-metrics-iperf3.sh

```diff
@@ -100,6 +106,48 @@ function iperf3_jitter() {
 	echo "Finish"
 }
 
+
+# Test/measure the bandwidth between the interconnection of a server/client.
+# A container takes the role of server and the host takes the role of client.
+function iperf3_host_container() {
+	local test_name="network host to container"
+
+	# Checks iperf3 tool installed in host
+	local cmds=("iperf3")
+	check_cmds "${cmds[@]}"
+
+	local server_command="mount -t ramfs -o size=20M ramfs /tmp && iperf3 -s -V"
+	local server_address=$(start_server "$image" "$server_command" "$server_extra_args")
+	local init_time=1
+
+	# Verify server IP address
+	if [ -z "$server_address" ];then
+		clean_env
+		die "server: ip address no found"
+	fi
+
+	# This time is in order to wait the container initialization
+	sleep $init_time
+	result=$(iperf3 -c $server_address -t $transmit_timeout)
```

Comment:
We need to run the tests in both direction.
The RX direction Host sending to VM
```+	result=$(iperf3 -c $server_address -t $transmit_timeout)```
and the TX direction VM sending to host

```result=$(iperf3 -c $server_address -t $transmit_timeout) -R```

Also add multiqueue tests where you run the test with -P X.
Where X is 2,4,8

Reply:
Hi @mcastelino thanks for feedback, I will add that